bigframes/dataframe.py

```diff
@@ -734,7 +734,7 @@ def _apply_binop(
         how: str = "outer",
         reverse: bool = False,
     ):
-        if isinstance(other, (float, int, bool)):
```

Comment:
I feel like just adding another type here is a narrow fix - what we really want to do is determine - is this type interpretable as a supported scalar. This could include datatime objects, decimal, etc as well. We should probably have a single definition of this somewhere

Reply:
Added a LOCAL_SCALAR_TYPES constant, all types are from infer_literal_type function, should be supported.